AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1456,6 +1456,7 @@ - (void)_insertSubnode:(ASDisplayNode *)subnode atSubnodeIndex:(NSInteger)subnod
         // If we can use view API, do. Due to an apple bug, -insertSubview:atIndex: actually wants a LAYER index, which we pass in
         if (canUseViewAPI && sublayerIndex != NSNotFound) {
           [_view insertSubview:subnode.view atIndex:sublayerIndex];
+          [(_ASDisplayView *)_view setAccessibleElements:nil];
```

Comment:
Should we put these in the view, using:

- (void)didAddSubview:(UIView *)subview;
- (void)willRemoveSubview:(UIView *)subview;

Instead?  Related question, does our current implementation work if the user directly adds a UIView to the hierarchy without a node attached to it?